From 0eb594296597e45011d8e4090244597f76d06d53 Mon Sep 17 00:00:00 2001
From: Scott Moser <scott.moser@chainguard.dev>
Date: Fri, 25 Oct 2024 12:55:29 -0400
Subject: [PATCH] Try bumping recursive-open-struct

recursive-open-struct moved forward to 1.3 on 2024-10-03 and then
immediately forward to 2.0 which does include this entry in changelog:

  * BREAKING: Restore #72, which is an API-breaking change because it changes
    what data is serialized with marshalling.

I proposed https://github.com/ManageIQ/kubeclient/pull/640 upstream
to see if it is plausible to do this versus adding
ruby3.2-recursive-open-struct-1.3 .

---
 kubeclient.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kubeclient.gemspec b/kubeclient.gemspec
index 4df0014..f1b9911 100644
--- a/kubeclient.gemspec
+++ b/kubeclient.gemspec
@@ -37,6 +37,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'jsonpath', '~> 1.0'
   spec.add_dependency 'rest-client', '~> 2.0'
-  spec.add_dependency 'recursive-open-struct', '~> 1.1', '>= 1.1.1'
+  spec.add_dependency 'recursive-open-struct', '> 1.1.1', '< 3.0'
   spec.add_dependency 'http', '>= 3.0', '< 6.0'
 end

-- 
2.47.0

